Makefile

```diff
@@ -29,6 +29,7 @@ run: certs
 		-e HTTPS_KEY_FILE=/certs/tls.key -e HTTPS_CERT_FILE=/certs/tls.crt \
 		-e ASSISTED_SERVICE_SCHEME=${ASSISTED_SERVICE_SCHEME} -e ASSISTED_SERVICE_HOST=${ASSISTED_SERVICE_HOST} \
 		-e REQUEST_AUTH_TYPE=${REQUEST_AUTH_TYPE} \
+		-e RHCOS_VERSIONS='${RHCOS_VERSIONS}' \
```

Comment:
:man_shrugging: Not sure it matters, but this variable isn't being added in this PR so I'd rather change it in a follow-up.

Reply:
Sure, can be renamed later on if we want. The logic behind using a generic OS naming was just to avoid being too specific with RHCOS, as we might support other distributions.

Reply:
Yup, I totally understand the reasoning, I just don't think the change belongs in this PR.